pyface/fields/tests/test_combo_field.py

```diff
@@ -1,163 +1,114 @@
-from __future__ import absolute_import
+# Copyright (c) 2019, Enthought, Inc.
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in enthought/LICENSE.txt and may be redistributed only
+# under the conditions described in the aforementioned license.  The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+# Thanks for using Enthought open source!
+
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+from six import text_type
 
 from traits.testing.unittest_tools import unittest
 
-from pyface.action.api import Action, MenuManager
 from pyface.image_resource import ImageResource
 from ..combo_field import ComboField
+from .field_mixin import FieldMixin
 
 
-class TestComboField(unittest.TestCase):
+class TestComboField(unittest.TestCase, FieldMixin):
 
-    def test_combo_field(self):
-        widget = ComboField(
-            parent=None,
+    def setUp(self):
+        FieldMixin.setUp(self)
```

Comment:
Will this not be called automatically?

Reply:
No, because `unittest.TestCase` has an empty `setUp` that doesn't call `super`.  :(

However, I could fix by swapping the order of the superclasses, which would be cleaner.

Reply:
How delightful! I'll leave it to you if you want to change it...